gammapy/estimators/points/tests/test_core.py

```diff
@@ -241,6 +241,19 @@ def test_plot(self, flux_points):
         with mpl_plot_check():
             flux_points.plot(ax=ax)
 
+    def test_plot_custom_y_units(self, flux_points):
+        fig = plt.figure()
+        ax = fig.add_axes([0.2, 0.2, 0.7, 0.7])
+        ax.xaxis.set_units(u.eV)
+
+        yunit = u.Unit("TeV cm-2 s-1")
+        ax.yaxis.set_units(yunit)
+
+        with mpl_plot_check():
+            ax = flux_points.plot(ax=ax, sed_type="e2dnde")
+
+        assert ax.yaxis.units == yunit
```

Comment:
this test does not fail on main branch. Is there really a problem setting custom units ?